Modules/_testcapi/unicode.c

```diff
@@ -110,6 +378,46 @@ unicode_fromobject(PyObject *self, PyObject *arg)
     return PyUnicode_FromObject(arg);
 }
 
+/* Test PyUnicode_InternInPlace() */
+static PyObject *
+unicode_interninplace(PyObject *self, PyObject *arg)
+{
+    NULLABLE(arg);
+    Py_XINCREF(arg);
+    PyUnicode_InternInPlace(&arg);
+    return arg;
+}
+
+/* Test PyUnicode_InternFromString() */
+static PyObject *
+unicode_internfromstring(PyObject *self, PyObject *arg)
+{
+    const char *s = NULL;
+    Py_ssize_t size = -100;
```

Comment:
Can't you leave them uninitialized? Compilers complain about uninitialized values? Stupid compilers :-) I mean, this test is not about testing PyArg_Parse() :-) Well, it's just a remark, your code is fine.

Reply:
Removed. I think it was just copied from other place.